### ISSUE TITLE
分享两个训练好的synthesizer模型

### ISSUE THREAD

**Issue by wrk226** [babysor/MockingBird#289]
俩模型都是用最新的代码来训练的，不需要切换回0.0.1
第一个模型synthesizer-merged_110k，是在代码支持的四个数据集（aidatatang_200zh，magicdata，aishell3，data_aishell）上联合训练的。learning rate=0.001无衰减，batch size=128，iteration=110k。
第二个模型synthesizer-zhvoice_170k，是在[zhvoice](https://github.com/fighting41love/zhvoice)这个数据集上训练的。learning rate=0.001无衰减，batch size=128，iteration=170k。
两个模型我都已经测试过是可用的，不过第一个似乎比第二个好一点。我觉得问题是出在vocoder上，因为我现在用的vocoder(wavernn)并不是在zhvoice数据集上训练的，而我也懒得再训练一个vocoder。不过用hifigan的话效果倒是差不多，不过音色又有明显不同，挺有意思的。
关于训练，我这俩模型还是可以进一步优化的，现在第一个模型的loss是在0.24左右，第二个是在0.22左右，不过花的时间太久我就懒得继续训练了。
下面是下载链接：
[百度云](https://pan.baidu.com/s/1Gt2MQydfrreBi4htYhSUCQ)  密码: ir90
[Google Drive](https://drive.google.com/drive/folders/10LDxmZOto9ehPbZgTyvY2NzPEjHS4qdG?usp=sharing)

**Comment by zhenzhendahu** [comment]
赞啊！感谢大佬，在下载了

**Comment by xinranxi** [comment]
赞啊，感谢感谢，家里破电脑没法训练，感谢模型

**Comment by amzsowhat** [comment]
感谢大佬！

**Comment by babysor** [comment]
手动 👍

**Comment by Battle-l** [comment]
感谢感谢

**Comment by wrk226** [comment]
> gzip: stdin: invalid compressed data--format violated

解压时候没遇到任何问题

**Comment by flysmart** [comment]
感谢分享！

**Comment by SunchinSekian** [comment]
为什么我第二个弄出来全是电流声，是有什么地方没配置好吗

**Comment by SunchinSekian** [comment]
感谢大佬，这个模型非常好

**Comment by x18-1** [comment]
新模型，新气象。感谢大佬

**Comment by EbenZhang** [comment]
我用main branch+这个synthesizer, 结果全是杂音/电流音
用别的 synthesizer 是好的.
请问大佬是不是我的打开方式不对?

以下是console输出:
```
Loaded encoder "pretrained.pt" trained to step 1594501
Synthesizer using device: cpu
Building hifigan
Loading 'vocoder\saved_models\pretrained\g_hifigan.pt'
Complete.
Removing weight norm...
Trainable Parameters: 0.000M
Loaded synthesizer "synthesizer-zhvoice_170k.pt" trained to step 170000
+----------+---+
| Tacotron | r |
+----------+---+
|   170k   | 2 |
+----------+---+

Read ['欢迎使用工具箱', '现已支持中文输入']
Synthesizing ['huan1 ying2 shi3 yong4 gong1 ju4 xiang1', 'xian4 yi3 zhi1 chi2 zhong1 wen2 shu1 ru4']

| Generating 1/1


Done.
```

**Comment by mikasa999** [comment]
还未下载，先谢为敬
